railties/lib/minitest/rails_plugin.rb

```diff
@@ -36,6 +36,10 @@ def record(result)
       @results << result
     end
 
+    def passed?
```

Comment:
Why we need this?

Reply:
Because the default implementation returns `false`, so `--profile` actually cause the test process to `exit 1`

https://buildkite.com/rails/rails/builds/104909#018da72b-0790-48d8-88c5-5ecfee2d322d/1453

Reply:
But should not it return `true` only if all tests passed?

Reply:
No, because there is more than one.

We have `CompositeReporter.new(regular_reporter, profile_reporter)`, so all of them need to pass for the suite to exit 0: https://github.com/minitest/minitest/blob/d00cc0d025ad0f79efb1dc7540c27aefa13d2d5f/lib/minitest.rb#L933-L935

So by always returning `true` here, I simply make it not have an impact on the result at all.